rosidl_generator_tests/package.xml

```diff
@@ -18,12 +18,16 @@
 
   <buildtool_depend>ament_cmake</buildtool_depend>
 
+  <test_depend>action_msgs</test_depend>
   <test_depend>ament_cmake_gtest</test_depend>
   <test_depend>ament_lint_auto</test_depend>
   <test_depend>ament_lint_common</test_depend>
+  <test_depend>ament_index_python</test_depend>
+  <test_depend>python3-jsonschema</test_depend>
```

Comment:
CI isn't doing `rosdep install` for deps, so test is failing to find `jsonschema` - it is not currently a dependency of the `ros2.repos` core. It is a valid existing rosdep key. How do we feel about adding this new dependency? I think it's good to have _and test_ the schemas.

If we do think it's acceptable to have it, do I need to make a PR against https://github.com/ros2/ci to fix the build?

Reply:
So it looks like `python3-jsonschema` is available in all of our Tier 1 and Tier 2 Linux platforms, so we can depend on it.  The open question will be Windows, where we will have to pip install it.

In order to add this in, then, we'll have to do the following:

1. Add a line to https://github.com/ros2/ci/blob/3659565a6f32ab293d33520f937360b0085e05a4/linux_docker_resources/Dockerfile#L48-L86 to install it
2. Add a line to https://github.com/ros2/ci/blob/3659565a6f32ab293d33520f937360b0085e05a4/linux_docker_resources/Dockerfile-RHEL#L78-L171 to install it
3. Add a line to https://github.com/ros-infrastructure/ros2-cookbooks/blob/fe4c0f7cbe8055ab7146ab9b9972a98a09189c4a/cookbooks/ros2_windows/recipes/pip_installs.rb#L1-L20 to install it on Windows
4. Update https://github.com/ros2/ros2_documentation/blob/7cd078439bcf69c0c8366e8971201d12e3a9fc5f/source/Installation/_Windows-Install-Prerequisites.rst?plain=1#L127 to add it to the list.

That is most of the places I can think of.  Note that all of this can (and most *must*) be done before this PR goes in, so we can start on that right now if you like.

Reply:
OK I have opened 
        * https://github.com/ros2/ci/pull/695
        * https://github.com/ros-infrastructure/ros2-cookbooks/pull/52
        * https://github.com/ros2/ros2_documentation/pull/3356